src/resources/arranger_es_metadata.json

```diff
@@ -1213,6 +1213,18 @@
             "type": "object",
             "primaryKey": false
           },
+          {
+            "unit": null,
+            "displayValues": {},
+            "quickSearchEnabled": false,
+            "field": "file_id",
```

Comment:
for reference, this is the mapping used for these fields in ES: https://github.com/icgc-argo/files-service/blob/develop/src/file-centric-index-mapping.json

according to this, it should be nested under file, so `file.file_id`

Reply:
nevermind, I misread the nesting. what you have here is correct.